dashboard/dashboard.html

```diff
@@ -980,6 +980,7 @@
 	logSegment.appendChild(h("div", attrs, [
 		h('span', null, " IGNOR "),
 		h('a', {"href": data["url"], "className": "ignore"}, data["url"]),
+		data["source"],
```

Comment:
Also, a question for @ivan: is this the intended use of strings in `appendChild`?  It looks like it is, but I haven't gone too far in-depth on the dashboard code.